modules/ui/sources/window.es

```diff
@@ -121,6 +121,9 @@ export default class {
 
     }.bind(this));
 
+    //mock default FF function
+    this.popup.enableOneOffSearches = function() {}
```

Comment:
@past tried that but we still get an error in the console at startup : 
TypeError: this.popup.enableOneOffSearches is not a function[Learn More]  urlbarBindings.xml:879:11